Tests in `main` currently broken

```
$ cargo test --release --manifest-path ./host/Cargo.toml --features central,gatt,peripheral,scan,security --lib -- --nocapture
   Compiling syn v2.0.114
   Compiling ff v0.13.1
   Compiling crypto-bigint v0.5.5
   Compiling group v0.13.0
   Compiling trouble-host v0.5.1 (/home/ubuntu/trouble.fork/host)
   Compiling elliptic-curve v0.13.8
   Compiling primeorder v0.13.6
   Compiling p256 v0.13.2
   Compiling darling_core v0.20.11
   Compiling thiserror-impl v2.0.18
   Compiling futures-macro v0.3.31
   Compiling futures-util v0.3.31
   Compiling thiserror v2.0.18
   Compiling unescaper v0.1.8
   Compiling darling_macro v0.20.11
   Compiling tokio-macros v2.6.0
   Compiling darling v0.20.11
   Compiling serialport v4.7.3
   Compiling tokio v1.49.0
   Compiling mio-serial v5.0.6
   Compiling embassy-executor-macros v0.7.0
   Compiling embassy-executor v0.9.1
   Compiling futures-executor v0.3.31
   Compiling futures v0.3.31
   Compiling trouble-host-macros v0.3.0 (/home/ubuntu/trouble.fork/host-macros)
   Compiling tokio-serial v5.4.5
   Compiling embedded-io-adapters v0.7.0
error[E0063]: missing field `last_handle_in_group` in initializer of `attribute::AttributeData<'_>`
    --> src/attribute.rs:1238:13
     |
1238 |             AttributeData::Service { uuid: GAP.into() },
     |             ^^^^^^^^^^^^^^^^^^^^^^ missing `last_handle_in_group`

error[E0063]: missing field `last_handle_in_group` in initializer of `attribute::AttributeData<'_>`
    --> src/attribute.rs:1296:13
     |
1296 |             AttributeData::Service { uuid: GATT.into() },
     |             ^^^^^^^^^^^^^^^^^^^^^^ missing `last_handle_in_group`

error[E0063]: missing field `last_handle_in_group` in initializer of `attribute::AttributeData<'_>`
    --> src/attribute.rs:1375:13
     |
1375 |             AttributeData::Service {
     |             ^^^^^^^^^^^^^^^^^^^^^^ missing `last_handle_in_group`

For more information about this error, try `rustc --explain E0063`.
error: could not compile `trouble-host` (lib test) due to 3 previous errors
```

Expectation:

- I can build `main` with any combination of features

Actual:

- fails, as seen above

The reason this got past CI is that it currently has pretty [pathetic tests for tests](https://github.com/embassy-rs/trouble/blob/main/ci.sh#L50-L52) - only using default features.  Likely, the glitch is introduced by some combination - I will have a look.

--- a/ci.sh
+++ b/ci.sh
@@ -48,5 +48,6 @@ cargo batch \
 cargo fmt --check --manifest-path ./host/Cargo.toml
 cargo clippy --manifest-path ./host/Cargo.toml --features gatt,peripheral,central
 cargo test --manifest-path ./host/Cargo.toml --lib -- --nocapture
+cargo test --manifest-path ./host/Cargo.toml --features central,gatt,peripheral,scan,security --lib -- --nocapture
 cargo test --manifest-path ./host/Cargo.toml --no-run -- --nocapture
 cargo test --manifest-path ./examples/tests/Cargo.toml --no-run -- --nocapture

--- a/host/src/attribute.rs
+++ b/host/src/attribute.rs
@@ -1235,7 +1235,10 @@ mod tests {
         // GAP service (handles 0x001 - 0x005)
         table.push(Attribute::new(
             PRIMARY_SERVICE.into(),
-            AttributeData::Service { uuid: GAP.into() },
+            AttributeData::Service {
+                uuid: GAP.into(),
+                last_handle_in_group: 0x05,
+            },
         ));
 
         let expected = 0xd4cdec10804db3f147b4d7d10baa0120;
@@ -1293,7 +1296,10 @@ mod tests {
         // GATT service (handles 0x006 - 0x000d)
         table.push(Attribute::new(
             PRIMARY_SERVICE.into(),
-            AttributeData::Service { uuid: GATT.into() },
+            AttributeData::Service {
+                uuid: GATT.into(),
+                last_handle_in_group: 0x0d,
+            },
         ));
 
         // Service changed characteristic
@@ -1374,6 +1380,7 @@ mod tests {
             PRIMARY_SERVICE.into(),
             AttributeData::Service {
                 uuid: CUSTOM_SERVICE.into(),
+                last_handle_in_group: 0x13,
             },
         ));
 

